### DISCUSSION TITLE
Stash won't start

### DISCUSSION THREAD

**Discussion by Bruckmi** [stashapp/stash#2533] [General]
Hi guys,

I was trying to change my folder for generated files and database yesterday, however I obviously messed something up and it started with initial setup, I was ok with it, however after the restart - Stash won't open, after clicking on stash-win.exe nothing is happening and http://localhost:9999/ is returning "This site can’t be reached". 

I have no idea if I am missing something. Any help on this?

Thanks a lot!

**Comment by WithoutPants** [comment]
Hi there.

Unfortunately, running stash in GUI mode (by clicking the exe) isn't currently outputting anything if it fails to start up. To get a better idea of what is going on, try running stash-win.exe from the command prompt. It should output an error message if it fails to start up.
